manifests/mpm.pp

```diff
@@ -97,14 +94,16 @@
         }
       }
 
-      if $mpm == 'prefork' {
-        include apache::mpm::disable_mpm_event
+      unless $mpm in ['itk', 'prefork'] {
```

Comment:
Is this right?
Your replacing the _if_ `$mpm` equals with an _unless_ `$mpm` equals.

Doesn't this now mean that `include apache::mpm::disable_mpm_prefork` won't be run if $mpm equals either of the two, where before it was run.